### PR TITLE
Revert reqwest, futures, tokio upgrade (revision 5502176e6)

### PR DESCRIPTION
**Pull request by Patrik-Stas**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: CI
 on:
   push:
     branches:
-      - main
+      - master
```
Comment (mirgee) [inline .github/workflows/main.yml:6]: Incorrect branch

File: agency_client/Cargo.toml [hunk 2]
```diff
@@ -16,7 +16,7 @@ lazy_static = "1.3"
 serde = "1.0.97"
 serde_json = "1.0.40"
 serde_derive = "1.0.97"
-reqwest = { version = "0.9.5" }
+reqwest = "0.9.5"
```
Comment (mirgee) [inline agency_client/Cargo.toml:19]: Doesn't seem to revert reqwest version.

File: agents/node/vcxagent-core/package.json [hunk 3]
```diff
@@ -69,6 +69,6 @@
     "winston": "^3.3.3"
   },
   "peerDependencies": {
-    "@hyperledger/node-vcx-wrapper": "^0.24.1"
+    "@hyperledger/node-vcx-wrapper": "^0.24.2"
```
Comment (mirgee) [inline agents/node/vcxagent-core/package.json:72]: I would prefer to have release PR separate as we usually do.

File: agency_client/Cargo.toml (hunk 2 above)
Comment (Patrik-Stas) [reply agency_client/Cargo.toml:19]: It doesn't revert reqwest specifically, but it does revert dependencies which previous upgrade of reqwest has caused bringing in in lock file. When the reqwest was previously downgraded from 11 to 10, then from 10 to 9, in neither of these downgrades we reverted dependencies brought in by the first `9->11` upgrade.

**Review by mirgee — APPROVED** [review]

**Pull request merged**